pkg/cmd/get_clustercompliancereport.go

```diff
@@ -50,7 +52,11 @@ func NewGetClusterComplianceReportsCmd(executable string, cf *genericclioptions.
 				return err
 			}
 			// generate compliance and compliance failure detail reports
-			complianceMgr := compliance.NewMgr(kubeClient, logger)
+			operatorConfig, err := etc.GetOperatorConfig()
+			if err != nil {
+				return fmt.Errorf("getting client config: %w", err)
```

Comment:
The "client" is not very clear probably:
```suggestion
				return fmt.Errorf("getting operator config: %w", err)
```